assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -143,11 +144,19 @@ function DashboardIdeasWidget( props ) {
 		triggerSurvey,
 	] );
 
+	if ( initialTotalNewIdeas === null && newIdeas?.length ) {
```

Comment:
I think we would want to still raise this even if `newIdeas` was an empty array.

Reply:
Good catch @aaemnnosttv . Updated.